### PR TITLE
Update aiohttp to 3.6.3; Downgrade multidict to 4.7.6, yarl to 1.5.1

### LINKED ISSUE #715: Revert multidict to 4.7.6
Commit 3a81bd28cb51d74f42a5a9ac20f106a623a6bd9d bumped _multidict_ to 5.0.0 despite being incompatible with the pinned version of _aiohttp_ (3.6.2, at this time but it is also incompatible with 3.6.3) which requires ``multidict>=4.5,<5.0``.

I'm happy to submit a PR for it if required but given it's a simple revert I'm not sure it's necessary.

### PR DESCRIPTION
**Pull request by lepaperwan**
This PR reverts 3a81bd28cb51d74f42a5a9ac20f106a623a6bd9d and 63926195169d79758f759c9e984a20e6a877bb5f in order to fix #707. Also closes #715.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,9 +8,9 @@ filelock==3.0.12
 idna==2.10
 importlib_resources==3.0.0; python_version < '3.7'
 lxml==4.6.1
-multidict==5.0.0
+multidict==4.7.6
```
Comment (cooperlees) [inline requirements.txt:11]:
```suggestion
multidict==4.7.6  # pyup: ignore
```

File: requirements.txt [hunk 2]
```diff
@@ -8,9 +8,9 @@ filelock==3.0.12
...
 packaging==20.4
 pyparsing==2.4.7
 setuptools==50.3.2
 six==1.15.0
-yarl==1.6.2
+yarl==1.5.1
```
Comment (cooperlees) [inline requirements.txt:16]:
```suggestion
yarl==1.5.1  # pyup: ignore
```

**Review by cooperlees — CHANGES_REQUESTED** [review]
Thanks for this. Was on my long list.

I think we'll need the pyup ignore comments or we'll just get back to exactly where we are :)

EDIT: I think PIP can handle comments in requirements files ...

**Comment by lepaperwan** [comment]
Yep pip can handle comments, and good catch on the pyup ignore :+1:

**Comment by cooperlees** [comment]
Rerunning - One job hit:
```
aiohttp.client_exceptions.ClientResponseError: 503, message='Service Unavailable', url=URL('https://pypi.org/pypi')
```

**Review by cooperlees — APPROVED** [review]

**Pull request merged**